appinventor/components-ios/src/ChartDataBase.swift

```diff
@@ -12,6 +12,8 @@ import DGCharts
   var _color: Int32 = AIComponentKit.Color.black.int32
   var _colors: [UIColor] = []
   var _label: String?
+  
+  var _dataLabelColor: Int32 = AIComponentKit.Color.black.int32
```

Comment:
Do we need to make this conditional on dark theme similar to what you did in Chart.swift?